VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/copy-quick-link/copy-quick-link.component.spec.ts

```diff
@@ -47,13 +47,15 @@ describe('CopyQuickLinkComponent', () => {
     });
 
     it('renders icon to copy to clipboard', () => {
+        component.ngOnInit();
+        fixture.detectChanges();
```

Comment:
I don't think these lines are needed since it's done in the before each. Was there a reason to add them?